cr17/nodes/cr17/mbed_interface.py

```diff
@@ -1,15 +1,287 @@
 #!/usr/bin/env python2
 import rospy
+from geometry_msgs.msg import Twist
+from cr17.msg import wheelData, scoopControl
+
+import os
+import sys
+
+#IMPORTANT!! These must be uncommented to work with the Mbed. Only uncommented for Travis-CLI testing
+#import usb.core
+#import usb.util
+ 
+from time import sleep
+import random
+
+'''
+Sends low-level commands(scoop, arm, linear velocity, angular velocity) to Mbed over USB.
+Also accepts speed values from Mbed for each wheel, and publishes to the wheel_speed topic. 
+
+This node must be run under su privelages. I've been doing it using sudo su, then the command to run the node. 
```

Comment:
If you make a launch file for this, and then on the node tag for this node add   'launch-prefix="sudo -E " '  it will handle the privileges and you won't need to run sudo su first.